nasa-gibs/worldview
# Add 2 Ozone L2 layers

## Description

Fixes #WV-3196.

Add in two Ozone layers

- TEMPO_L2_Ozone_Column_Amount_Granule
- TEMPO_L2_Ozone_Cloud_Fraction_Granule

## How To Test

[Provide whatever information a reviewer might need to know in order to verify that the changes made in this PR are working as expected. If there are special build steps that need to be taken in order to get these changes to run (building while on a separate branch, running `npm ci`, etc) include them here.]

- For bugfixes: What steps need to be taken in the UI to verify the bug is fixed?
- For enhancements and features: What is the expected functionality being added/modified? How can a reviewer verify this?

1. Open with [these URL parameters](http://localhost:3000/?v=-275.6887494997544,-29.00778373364843,17.704289251940537,104.50775874267788&z=4&ics=true&ici=5&icd=10&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m(hidden),TEMPO_L2_Ozone_Cloud_Fraction_Granule(count=1),TEMPO_L2_Ozone_Column_Amount_Granule(hidden,count=1),Land_Water_Map(opacity=0.8),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-05-29-T14%3A50%3A00Z)
2. Check to make sure the two layers display on the map, layer descriptions are correct, etc. 
3. Verify expected result


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
